tools/generator/templates/player/cs/Makefile-cs.jinja2

```diff
@@ -0,0 +1,21 @@
+{# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (c) 2020 Association Prologin <association@prologin.org>
+#}
+{% extends "includes/Makefile-language.jinja2" %}
+
+{% block lang %}cs{% endblock %}
```

Comment:
@seirl should I switch to `csharp` here as well ? Not sure what stechec expects from the content of the `_lang` file, it was `cs` for the ruby generator

Reply:
let's stick to `cs` for now in `_lang`